package.json

```diff
@@ -161,7 +161,6 @@
     "acorn-walk": "^8.1.1",
     "arg": "^4.1.0",
     "diff": "^4.0.1",
-    "glob": "^10.0.0",
```

Comment:
The changes of this commit are actually fixing the previous commit, so it shouldn't be a separate commit.